tests/rest/test_single_topology_rest.py

```diff
@@ -111,11 +113,13 @@ def test_single_topology_rest_vacuum(mol_pair, temperature_scale_interpolation_f
     else:
         assert energy_scale < 1.0
 
+        # check that a proper subset of ligand-ligand nonbonded interactions are scaled
+        assert not np.isclose(U_nonbonded, U_nonbonded_ref)
+        assert not np.isclose(U_nonbonded, energy_scale * U_nonbonded_ref)
+
         if has_rotatable_bonds or has_aliphatic_rings:
             assert 0 < len(st_rest.candidate_propers)
-
-            if energy_scale < 1.0:
-                assert U_proper < U_proper_ref
+            assert U_proper < U_proper_ref
```

Comment:
Is the energy strictly lower in the REST case? eg. if we have a torsion potential that was strictly between `[-10, 0]`, removing it would increase the energy instead?

Reply:
Good point; this assumes that k is positive, which may not be strictly true in our parameterization? (It does seem to be for the examples tested here)